timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -869,18 +856,21 @@ void declare_bound_potential(py::module &m) {
             "execute_fixed",
             [](timemachine::BoundPotential &bp,
                const py::array_t<double, py::array::c_style> &coords,
-               const py::array_t<double, py::array::c_style> &box) -> py::tuple {
+               const py::array_t<double, py::array::c_style> &box) -> const py::array_t<uint64_t, py::array::c_style> {
```

Comment:
I can't remember why we chose for this to return a numpy array of size 1 as opposed a scalar - might be good to standardize on this later.

Reply:
I think the reason for this, as I tried to just return `uint64` is that it returns a python int, which seemed to break the conversion. This way it is a numpy array with a fixed type that is maintained through the wrapping layer.

Reply:
Ah good call, thanks.